resources/scripts/downloader.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+DEST=$1
+
+if [ "$DEST" == "" ]; then
+  echo "You must specify a destination directory."
```

Comment:
question(non-blocking): Are this echos readable to someone?

Reply:
I believe they should be in the cloudwatch logs